2748. Number of Beautiful Pairs/solution.py

```diff
@@ -0,0 +1,39 @@
+class Solution:
+    # Given a list of nums, it returns the number of beautiful pairs. A pair of
```

Comment:
nit: what is it?

Reply:
you should just say, "this function" or just "return the number..."